app/views/users/_form.html.haml

```diff
@@ -1,38 +1,54 @@
-%table.form{:width => "70%"}
-  %tr
-    %th First Name:
-    %td.top= text_field :user, :firstName, value: @user.firstname
-  %tr
-    %th Last Name:
-    %td= text_field :user, :lastName, value: @user.lastname
-  %tr
-    %th Account Name:*
-    %td= text_field :user, :username, value: @user.username
-  %tr
-    %th Email Address:*
-    %td= text_field :user, :email, value: @user.email
-  %tr
-    %th Re-enter Email Address:*
-    %td= text_field :user, :email_confirmation, value: @user.email_confirmation
-  %tr
-    %th Password:*
-    %td= password_field :user, :password, :autocomplete => "off"
-  %tr
-    %th Re-enter Password:*
-    %td= password_field :user, :password_confirmation, :autocomplete => "off"
-  - if using_captcha?
-    %tr
-      %th
-      %td= recaptcha_tags
-  - unless $ANNOUNCE_LIST.nil? || $ANNOUNCE_LIST.empty?
-    %tr
-      %td{:colspan => "2", :style => "text-align: right;"}
-        %div{:style => "float: right; margin-left: 10px; position: relative; top: 5px;"}
-          %input#user_register_mail_list{:checked => "checked", :name => "user[register_mail_list]", :type => "checkbox", :value => "1"}/
-        %div{:style => "overflow: hidden; line-height: 12px;"}
-          %label{:for => "user_register_mail_list"}
-            Register for the #{$SITE} announcements email list
-            %br/
-            %span{:style => "font-size: x-small;"} (approximately two emails per month -- confirmation via email required)
-  %tr
-    %td{:colspan => "2", :style => "text-align: right;"}= submit_tag "Create"
+.d-flex.justify-content-center
+  .register-form
+    .d-flex.justify-content-between
+      %a{:href => "javascript:javascript:history.go(-1)"}
+        %svg.register-arrowback{:fill => "none", :height => "12", :viewbox => "0 0 15 12", :width => "15", :xmlns => "http://www.w3.org/2000/svg"}
+          %path{:d => "M5.64176 1L1 6L5.64176 11M14 6H1.13", :stroke => "black", "stroke-linecap" => "round", "stroke-linejoin" => "round", "stroke-miterlimit" => "10", "stroke-width" => "1.5"}
+      .register-title-container
+        %h2.register-title
+          Create new account
+        %hr#register-title-line/
+      %div
+    .register-double-input
+      .register-first-input
+        %p.register-input-title
+          First name
+          %font{:color => "red"} *
+        = text_field :user, :firstName, value: @user.firstname, class: "register-input-short"
+      %div
+        %p.register-input-title
+          Last name
+          %font{:color => "red"} *
+        = text_field :user, :lastName, value: @user.lastname, class: "register-input-short"
+    %p.register-input-title
+      Username
+      %font{:color => "red"} *
+    = text_field :user, :username, value: @user.username, class: "register-input-long"
+    %p.register-input-title
+      ORCIDE ID
+      %font.register-optional (optional)
+    %img.register-input-icon{:src => "#{asset_path("orcide.svg")}"}/
```

Comment:
We have this error: `The asset "orcide.svg" is not present in the asset pipeline.`

Add this svg in this PR